src/go/configmanager/flags/flags.go

```diff
@@ -134,6 +134,20 @@ var (
 	TranscodingPreserveProtoFieldNames      = flag.Bool("transcoding_preserve_proto_field_names", false, "Whether to preserve proto field names for grpc-json transcoding")
 	TranscodingIgnoreQueryParameters        = flag.String("transcoding_ignore_query_parameters", "", "A list of query parameters(separated by comma) to be ignored for transcoding method mapping in grpc-json transcoding.")
 	TranscodingIgnoreUnknownQueryParameters = flag.Bool("transcoding_ignore_unknown_query_parameters", false, "Whether to ignore query parameters that cannot be mapped to a corresponding protobuf field in grpc-json transcoding.")
+
+	BackendRetryOns = flag.String("backend_retry_ons", "reset,connect-failure,refused-stream",
```

Comment:
How to disable retry?  set it to empty value?

Reply:
It can be done by empty `RetryOns` or 0 `retryNum`. Put it in the flag explanation.